blog/articles/orchestrating-cloud-events-with.md

```diff
@@ -0,0 +1,194 @@
+
+Until this point, this is just a normal Knative application and, because you are using Knative Eventing which natively supports CloudEvents, you can integrate more tools to the ecosystem as you will see in the next section.
+
+## Visualize / Understand
+
+Without changing anything inside the application code or setup, you can tap into the event stream and map the events that are meaningful for your customer journey and visualize them using [Zeebe](http://zeebe.io) and [Camunda Operate](https://camunda.com/products/cloud/):
+
+![Knative and Zeebe](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-service-knative-zeebe.png?raw=true)
+
+You can start simply by creating a workflow model that consumes the events emitted by the application to report status, as the following model shows:
+
+![Workflow model v1](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-v1.png?raw=true)
+
+This simple workflow model waits for the application events and moves the flow forward when each event arrives. And yes, you guessed it, it works with **CloudEvents** :).
+
+For each customer that joins the queue to buy tickets, a new workflow instance is created to keep track of them. Having these instances allows you to track individual customers and quickly understand where everyone is at a given point in time.
+
+![workflow model instances](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-v1-operate-1.png?raw=true)
+
+Zeebe comes with Camunda Operate, which allows you to see all this information in almost real-time and have a detailed audit trail about the workflow execution and the data associated with it.
+
+![Operate UI](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-v1-operate-2.png?raw=true)
+
+## Decorate / Enhance
+
+Now that you have the power of a workflow engine, you can leverage some of its features to decorate or enhance your applications. An ehnchanced version of the workflow model could look like this:
+
+![Workflow model v2](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-v2.png?raw=true)
+
+Now when the customer reaches the **Payment Sent** state, meaning that they have already made a reservation, the model wait for the **Payment Sent** CloudEvent to arrive.
+
+![timers](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-v2-timers.png?raw=true)
+
+As you can see above, two timer boundary events have been associated with the **Payment Sent** state. The bottom one, with dashed lines, is a non-interrupting event, meaning that it will not disrupt the actual flow, it will be just triggered in a fire-and-forget fashion. This result in the Payment Reminder being sent and, because it can be configured as a recurring timer, it will trigger every X amount of time (for the presentation this was set to 10 seconds).
```

Comment:
```suggestion
As you can see above, two timer boundary events have been associated with the **Payment Sent** state. The bottom one, with dashed lines, is a non-interrupting event, meaning that it will not disrupt the actual flow, it will be just triggered in a fire-and-forget fashion. This results in the Payment Reminder being sent and, because it can be configured as a recurring timer, it will trigger every X amount of time (for the presentation this was set to 10 seconds).
```